src/runtime/pkg/katautils/config.go

```diff
@@ -451,11 +451,13 @@ func (h hypervisor) getInitrdAndImage() (initrd string, image string, err error)
 
 	image, errImage := h.image()
 
-	if image != "" && initrd != "" {
+	if h.ConfidentialGuest && h.MachineType == "s390-ccw-virtio" {
```

Comment:
can you use `QemuCCWVirtio` in this package ?

Reply:
ah yes, we already import `virtcontainers` anyway